server/src/main/java/org/elasticsearch/common/settings/Setting.java

```diff
@@ -83,6 +82,11 @@
  * </pre>
  */
 public class Setting<T> implements ToXContentObject {
+    private static final String DEPRECATED_MESSAGE_TEMPLATE =
+        "[{}] setting was deprecated in Elasticsearch and will be removed in a future release. "
+            + "See the %s changes documentation for the next major version.";
+    private static final String DEPRECATED_WARN_MESSAGE = String.format(Locale.ROOT, DEPRECATED_MESSAGE_TEMPLATE, "deprecation");
```

Comment:
'deprecation changes' doesn't make much sense, how about just 'See the deprecation documentation...' instead?

Reply:
There's still some references to the old wording (and `String.format(Locale.ROOT`) in this PR eg in ESTestCase